### PR TITLE
Removed unused AWeber Templates. Added a new single template for AWeb…

### PR DESCRIPTION
**Pull request by davestys**
Removed unused AWeber Templates. Added a new single template for AWeber with multiple groups.
Needs review from IONOS, Google Domains, GoDaddy prior to installation please.

### CODE REVIEW

**Comment by pawel-kow** [comment]
@davestys this template as proposed in this PR is impossible, because it asks for CNAME on @ which would exclude usage on APEX domain and require setting `hostRequired` template property. You'd need at least to split into 2 templates - for APEX and for a subdomain. This is why the schema check is failing.
Also please consider if the groups of records correspond to a separate service which can be independently provisioned or deprovisioned without affecting others, or whether they constitute one service.
In the first case a separate template is recommended, in the latter one template with groupIds would be more suited.

File: aweber.com.email-web.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+       {
+         "groupId": "web",
+         "type": "CNAME",
+         "host": "www",
+         "pointsTo": "hosted-content.aweber.com",
+         "ttl": 3600
+       },
+       {
+         "groupId": "web-subdomain",
+         "type": "CNAME",
```
Comment (pawel-kow) [inline aweber.com.email-web.json:85]: this record is failing the schema without `hostRequired` template property set

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: aweber.com.email-web.json (hunk 1 above)
Comment (davestys) [reply aweber.com.email-web.json]: I created a separate template for the subdomain situation. Decision is made to maintain this as one service where one or more groups can be applied at once.

**Comment by davestys** [comment]
Ok, looks like the template is valid @pawel-kow . Thanks for the advice. Let me know if you see any further concerns.

**Review by pawel-kow — APPROVED** [review]
All good here.

**Comment by pawel-kow** [comment]
@rphilip can you revuew for GoDaddy? Not sure who from Google Domains. @davestys do you have a contact person there to ask? I will post on Slack on this anyway.

**Comment by davestys** [comment]
> @davestys do you have a contact person there to ask? I will post on Slack on this anyway.

Yes, Xiaotong Chen. I will send an email asking for review as well.

**Review by wdbaker-godaddy — APPROVED** [review]
No problems found.

**Pull request merged**